modelina-website/src/components/playground/Playground.tsx

```diff
@@ -96,6 +96,10 @@ class Playground extends React.Component<
     phpIncludeDescriptions: false,
     phpNamespace: 'AsyncAPI/Models',
     cplusplusNamespace: 'AsyncapiModels',
+    pythonPackageName: 'asyncapi.models',
```

Comment:
I cant find any reference to python package name option or why I included it in the list of the issue 🤔 https://github.com/asyncapi/modelina/blob/1c9e647c5e3b9a3bf4482971456c29e8e65980ca/src/generators/python/PythonGenerator.ts#L31

We can safely remove python here, sorry for that!